### PR TITLE
Finish migration from nacho.yml to windspigot.yml

### PR DESCRIPTION
**Pull request by ghost**
## Description
Moves all of NachoConfig & NachoWorldConfig to WindConfig, with a new WindWorldConfig and migration.
This also replaces the Sugarcane comment implementation with SnakeYaml's.
This may break some plugins which use SnakeYaml directly instead of Bukkit's implementation.

## Status
Migration works as expected, though the comments seeming don't work for whatever reason.

### CODE REVIEW

File: WindSpigot-Server/src/main/java/ga/windpvp/windspigot/config/WindSpigotConfig.java [hunk 1]
```diff
@@ -81,9 +77,44 @@ public static void init(File configFile) {
 		}
 		
 	}
-	
-	// Not private as the config is read by calling all private methods with 0 params
-	static void makeReadable() {
+
+	private static final Map<String, String> oldToNewConfigNames = new HashMap<String, String>() {{
+		put("settings.fixed-pools.size", "settings.async.tnt.pool-size");
```
Comment (windcolor-dev) [inline WindSpigot-Server/src/main/java/ga/windpvp/windspigot/config/WindSpigotConfig.java]:
```suggestion
		put("settings.fixed-pools.size", "settings.async.explosions.enable");
```
This is the new path

Comment (ghost) [reply WindSpigot-Server/src/main/java/ga/windpvp/windspigot/config/WindSpigotConfig.java]: I am incorrect, however it's not that but instead `settings.async.explosions.threads`
```suggestion
		put("settings.fixed-pools.size", "settings.async.explosions.threads");
```

Comment (windcolor-dev) [reply WindSpigot-Server/src/main/java/ga/windpvp/windspigot/config/WindSpigotConfig.java]: Oh XD wrong setting

**Comment by ghost** [comment]
I currently have a lot to do and have not much free time to work on WindSpigot. As such, I will be closing this PR and will not be working on WindSpigot. If anybody would like to continue to this PR, everything is there, there is just a problem causing the comments to not load properly, which can be fixed by reverted to the old comment system, (though I had problems while doing this for some reason even though it works properly on the main branch) or getting Bukkit's new comment system to work on 1.8.

**Pull request closed without merging**